verification/curator-service/api/src/geocoding/geocoder.ts

```diff
@@ -0,0 +1,20 @@
+// A single result from a Geocoder.
+export interface GeocodeResult {
+    geometry: {
+        latitude: number;
+        longitude: number;
+    };
+    country: string;
+    // First administrative division (regions in the US, Länder in Germany, ...).
+    administrativeAreaLevel1: string | undefined;
+    // Second administrative division (county in the US, departments in France, ...).
+    administrativeAreaLevel2: string | undefined;
+    locality: string | undefined;
```

Comment:
Could you add a comment for locality too?

Reply:
this is going away as per Kelly's refactoring of other fields, I'll align those once she submits her other PR.